Okta.Core.Tests/Clients/Helpers.cs

```diff
@@ -155,5 +156,31 @@ internal static Models.Group CreateGroup(OktaClient oktaClient, TestGroup dbGrou
             return oktaGroup;
         }
 
+        internal static string GetRandomString()
+        {
+            const string AllowedChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz#@$^*()";
+            Random rng = new Random();
+
+            return RandomString(AllowedChars, 8, 16, rng);
+        }
+
+        private static string RandomString(
+            string allowedChars,
+            int minLength,

```

Comment:
Just curious - why do we need so much variability here? I.e. seems like all we really want is "give me a random string of length x". Not a big deal, but feels like we're overcomplicating this helper function that is only used in one place.